tuf/download.py

```diff
@@ -399,122 +380,27 @@ def _download_fixed_amount_of_data(connection, temp_file, required_length):
     raise
 
   else:
-    # This else block returns and skips closing the connection in the finally
-    # block, so close the connection here.
-    connection.close()
+    # This else block returns and skips closing the response in the finally
+    # block, so close the response here.
+    response.close()
```

Comment:
Not related to this PR, but this `finally` clause with the response.close is then useless. I don't think `finally` works like this at all...

Reply:
Santiago, that was fixed in another PR and will disappear here once it's rebased and the conflict is resolved (shortly).